aspnet/tutorials/publish-to-azure-webapp-using-vs.rst

```diff
@@ -1,47 +1,181 @@
-Publish to an Azure Web App using Visual Studio
-===============================================
+Deploy an ASP.NET Core web app to Azure using Visual Studio
+============================================================
 
-By `Erik Reitan`_
+By `Rick Anderson`_
 
-This article describes how to publish an ASP.NET web app to Azure using Visual Studio. 
 
-**Note:** To complete this tutorial, you need a Microsoft Azure account. If you don't have an account, you can `activate your MSDN subscriber benefits`_ or `sign up for a free trial`_.
+.. contents:: Sections:
+  :local:
+  :depth: 1
 
-.. _`activate your MSDN subscriber benefits`: http://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/?WT.mc_id=A261C142F
+Set up the development environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. _`sign up for a free trial`: http://azure.microsoft.com/pricing/free-trial/?WT.mc_id=A261C142F
+- `Install the latest Azure SDK for Visual Studio 2015 <http://go.microsoft.com/fwlink/?linkid=518003>`__. The SDK installs Visual Studio 2015 if you don't already have it.
 
+.. note:: The SDK installation can take more than 30 minutes if your machine doesn't have many of the dependencies.
```

Comment:
Do we really need this? Who is going to remove this if they update the installer to make it faster?